ctk/idp/src/main/kotlin/org/codice/compliance/web/sso/PostSSOTest.kt

```diff
@@ -60,7 +61,7 @@ class PostSSOTest : StringSpec() {
             }
         }
 
-        "POST AuthnRequest With Relay State Test" {
+        "POST AuthnRequest With Relay State Test (relay state verification from BindingVerifier)" {
```

Comment:
❓ What is the purpose of the statement inside of the parenthesis?

Reply:
We were adding comments in the test instead of using the test name to define what the test does